doc/source/whatsnew/v0.18.2.txt

```diff
@@ -51,6 +51,70 @@ New behaviour:
 
    In [2]: pd.read_csv(StringIO(data), names=names)
 
+.. _whatsnew_0182.enhancements.semi_month_offsets:
+
+Semi-Month Offsets
+^^^^^^^^^^^^^^^^^^
+
+Pandas has gained new frequency offsets, ``SemiMonthEnd`` ('SM') and ``SemiMonthBegin`` ('SMS').
+These provide date offsets anchored (by default) to the 15th and end of month, and 15th and 1st of month respectively.
+Optionally you can specify the ``day_of_month`` which will be used instead of the 15th each month. Anchoring suffixes
+are also supported for setting the `day_of_month``. Note that ``day_of_month`` must be 1<=day_of_month<=27 for
```

Comment:
you don't need this last sentence (the Note). That can be part of the doc-string.